### PR TITLE
Adding ML-Git Uninstall Section

### PR DESCRIPTION
**Pull request by gilMars**

### CODE REVIEW

**Review by Mun1z — APPROVED** [review]

**Review by lucasncabral — COMMENTED** [review]
I think adding a section on how to uninstall may end up polluting this file, maybe it's the case to create another file and just add the link to it, even in the final section 'Links'. It's a suggestion.

**Comment by gilMars** [comment]
> I think adding a section on how to uninstall may end up polluting this file, maybe it's the case to create another file and just add the link to it, even in the final section 'Links'. It's a suggestion.

Create uninstall section in another file is not intuitive, and uninstall process is aggregated with install process.

**Review by lucasncabral — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -36,6 +36,39 @@ Download ML-Git from repository and execute commands below:
...
+    ```
+  
+    *Root User*:
+    ```
+    sudo rm -rf /usr/local/lib/pythonX.Y/dist-packages/ml_git*
+    sudo rm -rf /usr/local/lib/pythonX.Y/dist-packages/ml-git*
+    sudo rm /usr/local/bin/ml-git
+    ```
+    **Note**: Change pythonX.Y to your python version. Example: python3.6
+
```
Comment (andersonpablosilva) [inline README.md:71]: Please, remove this section. We've updated the installation process and can keep the documentation simple by just recommending uninstalling using pip.

**Review by andersonpablosilva — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (gilMars) [reply README.md]: Removed in f957e701. Thanks a lot.

**Review by andersonpablosilva — APPROVED** [review]

**Pull request merged**